packages/cozy-stack-client/src/PermissionCollection.js

```diff
@@ -171,23 +206,25 @@ export const getPermissionsFor = (
 ) => {
   const { _id, _type } = document
   const verbs = options.verbs ? options.verbs : publicLink ? ['GET'] : ['ALL']
+  const name = options.name || 'files'
+  const collectionName = (options.name || '') + 'collection'
   // TODO: this works for albums, but it needs to be generalized and integrated
   // with cozy-client ; some sort of doctype "schema" will be needed here
-  return isFile(document)
+  return !isCollection(document)
```

Comment:
Nit: I think it is clearer when the condition is positive

```patch
- isCollection(document) ? otherStuff : stuff
+ isCollection(document) ? stuff : otherStuff
```

Reply:
:+1: